pkg/pipeline/transform/transform_network.go

```diff
@@ -189,15 +189,16 @@ func NewTransformNetwork(params config.StageParam) (Transformer, error) {
 
 	var servicesDB *netdb.ServiceNames
 	if needToInitNetworkServices {
+		pFilename, sFilename := jsonNetworkTransform.GetServiceFiles()
 		var err error
-		protos, err := os.Open(jsonNetworkTransform.ProtocolsFile)
+		protos, err := os.Open(pFilename)
 		if err != nil {
-			return nil, fmt.Errorf("opening %q: %w", jsonNetworkTransform.ProtocolsFile, err)
+			return nil, fmt.Errorf("opening protocols file %q: %w", pFilename, err)
 		}
 		defer protos.Close()
-		services, err := os.Open(jsonNetworkTransform.ServicesFile)
+		services, err := os.Open(sFilename)
 		if err != nil {
-			return nil, fmt.Errorf("opening %q: %w", jsonNetworkTransform.ServicesFile, err)
+			return nil, fmt.Errorf("opening services file %q: %w", sFilename, err)
```

Comment:
+1 for improving the error message

Reply:
I was able to find the error message in the e2e test of the PR that introduced the bug. But, despite that, the test passes.
https://github.com/netobserv/flowlogs-pipeline/actions/runs/3082582328/jobs/4982452400
![image](https://user-images.githubusercontent.com/37826670/191749995-9c3877a6-131c-4cb7-910d-84a7e012ef38.png)

Reply:
yeah seems like there's some improvement to bring to the e2e test too